scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -640,6 +665,47 @@ def test_magnitude_single_rotation():
     assert_allclose(result2, 0)
 
 
+def test_reduction_no_indices():
+    result = Rotation.identity().reduce(return_indices=False)
+    assert type(result) == Rotation
```

Comment:
Not super important, but `isinstance` is considered to be more idiomatic.

Reply:
Nice. I have fixed it.